src/main/resources/templates/modal/resetModal.html

```diff
@@ -1,16 +1,16 @@
 <html xmlns:th="http://www.thymeleaf.org" lang="en">
-<div class="modal-header">
-    <h5 class="modal-title" id="reset-modal">Reset Grouping</h5>
-    <a class="close" type="button" ng-click="cancelResetGroup()">
-        <span aria-hidden="true">&times;</span>
-    </a>
-</div>
-<div class="modal-body" id="modal-body">
+  <div class="modal-header">
+      <h5 class="modal-title" id="reset-modal">Reset Grouping</h5>
+      <a class="close" type="button" ng-click="cancelResetGroup()">
+          <span aria-hidden="true">&times;</span>
+      </a>
+  </div>
+  <div class="modal-body" id="modal-body">
     <p>
-        <span class="text-dark" th:text="#{screen.message.modal.reset.confirmation}"></span>
-        <span class="text-dark">{{listNames}} in the {{groupReset}} group?</span>
+      <span class="text-dark" th:text="#{screen.message.modal.reset.confirmation}"></span>
+      <span class="text-dark">{{listNames}} in the {{groupReset}} group?</span>
     </p>
-    <div class="container modal-table" tabindex="0">
+    <div class="container modal-table" tabindex="0" id="modalTableContainer">
```

Comment:
Was adding `id="modalTableContainer"` required to pass the ADA check? If not, this can be removed, thanks.